Documentation/contributing.rst

```diff
@@ -449,7 +449,6 @@ contain any characters after it.
 
 The Kubernetes tests support the following Kubernetes versions:
 
-* 1.6
 * 1.7
 
 By default, the Vagrant VMs are provisioned with Kubernetes 1.7. To run with any other
```

Comment:
We need to update these docs as well as we support 1.8 by default, now. This is outside the scope of this ticket, though.

Reply:
Filed https://github.com/cilium/cilium/issues/3105